app/helpers/member_helper.rb

```diff
@@ -0,0 +1,23 @@
+module MemberHelper
+  def image_tag_for(member, github_link: false)
+    options =
+      if github_link
```

Comment:
@adilsoncarvalho I thought to do this using three methods but the result was not too good, because we will need to call with different names, I prefered to keep in this way even :)